tests/test_usersettableparam.py

```diff
@@ -20,6 +26,7 @@ def setUp(self):
             "slider", value=123, min_value=100, max_value=200
         )
         self.slider_option_standalone = Slider(value=123, min_value=100, max_value=200)
+        self.static_text_option = StaticText("Hurr, Durr Im'a Sheep")
```

Comment:
Nice :)

Reply:
It's a reference to Linux Kernel 4.0: https://www.kernel.org/hurr-durr-ima-sheep.html.